File: homeworld_events/hwe-app/src/main/java/com/nastyhaze/homeworld/hwe_app/util/CommonUtils.java
package com.nastyhaze.homeworld.hwe_app.util;

import java.util.Objects;

/**
 *  Static Utility class for common application operations.
 */
public class CommonUtils {

    /**
     * Utility method for String validation.
     * Returns true: String contains a null value or only whitespace.
     * Returns false: String contains valid data.
     * @param str
     * @return boolean
     */
    public static boolean isNullOrBlankString(String str) {
        return Objects.isNull(str) || str.isBlank();
    }
}

File: homeworld_events/hwe_webapp/src/app/shared/util/common-utils.ts
/**
 * Utility method for string validation.
 * Returns true: String contains a null value or only whitespace.
 * Returns false: String contains valid data.
 * @param str
 * @returns boolean
 */
export function isNullorBlankString(str: string): boolean {
    return str == null || str.trim().length === 0;
}